cmd/agola/cmd/runlist.go

```diff
@@ -65,6 +65,21 @@ func printRuns(runs []*gwapitypes.RunResponse) {
 		fmt.Printf("%s: Phase: %s, Result: %s\n", run.ID, run.Phase, run.Result)
 		for _, task := range run.Tasks {
 			fmt.Printf("\tTaskName: %s, Status: %s\n", task.Name, task.Status)
+			printTaskDetails(run.ID, task.ID)
+		}
+	}
+}
+
+func printTaskDetails(runid, taskid string) {
+	gwclient := gwclient.NewClient(gatewayURL, token)
+	task, _, err := gwclient.GetRunTask(context.TODO(), runid, taskid)
+	if err == nil {
```

Comment:
We shouldn't ignore possible `GetRunTask` errors here, two options:

* Just exit with an error if any gateway call fails.
* Record the error and just print the error when printing the task details (something like: `failed to retrieve task information:   the error`)